xcode/Userscripts/ViewController.swift

```diff
@@ -90,6 +94,15 @@ class ViewController: NSViewController {
                     }
                     self.saveLocation.stringValue = url.absoluteString
                     self.saveLocation.toolTip = url.absoluteString
+                    SFSafariApplication.dispatchMessage(
+                        withName: "SAVE_LOCATION_CHANGED",
+                        toExtensionWithIdentifier: self.extensionID,
+                        userInfo: ["saveLocation": url.absoluteString.removingPercentEncoding ?? "??"]
+                    ) { error in // always be called
+                        if error != nil {
+                            debugPrint("Message attempted. Error info: \(String.init(describing: error))")
+                        }
+                    }
```

Comment:
Right now when the user changes the save location, the manifest still refers to the old save location content., but it would be nice to also run the manifest cleanup/update process once we know the save location was successfully changed. The [same code that runs on `popupInit`](https://github.com/quoid/userscripts/blob/9c7ec32f11a554a34c60962120164b3285c1cb9e/xcode/Userscripts%20Extension/Functions.swift#L1920).

This is not a blocker and I think this could be done in a different PR. I thought of this when reviewing and wanted to comment on it.